migrations/sql/15_pg_stat_monitor_09_histogram.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE metrics
+  ADD COLUMN `histogram_items` Array(String);
```

Comment:
I think Map or Nested would be better fit here, 
https://clickhouse.com/docs/en/sql-reference/data-types/map/, https://clickhouse.com/docs/en/sql-reference/data-types/nested-data-structures/nested/

Reply:
I see that clickhouse-go client doesn't support it, so ignore it.